download_all_mp3.py

```diff
@@ -15,13 +15,12 @@
 
 def download_mp3(word, url, dir_path):
     temp=requests.get(url).content
-    if "html" in str(temp[:15]): # <!DOCTYPE html> or <html>
-        raise Exception
-    else :
-        #print('ok')
+    if ((temp[1]>=224) and (temp[0]==255)):#1111-1111-111 mp3 first bit sync
```

Comment:
check the first bytes of to find out about file
mp3 files start with   11 bit sync 1111-1111-111